File: 1.Arrays/Java/MinMaxArray.java
public class MinMaxArray {

    public static void main(String[] args) {
        int[] arr = {10,6,70,89,3};

        int max = arr[0], min = arr[0];
        for (int i = 0; i < 5; i++)
        {
            if (arr[i] > max){
                max = arr[i];
            }
            if (arr[i] < min){
                min = arr[i];
            }        
        }
        System.out.print("\n Max = "+max+" \n Min = "+min);
    }
}

File: 1.Arrays/Kotlin/MinMaxArray.kt
fun main(){
    var arr = arrayOf(50,40,1,35,90)
    var max = arr[0]
    var min = arr[0]
    for (i in 0..arr.size-1) {
        if (arr[i] > max){
            max = arr[i];
        }
        if (arr[i] < min){
            min = arr[i];
        }    
    }
    print("\n Max = "+max+" \n Min = "+min)
}